app/code/Magento/Customer/view/frontend/templates/form/login.phtml

```diff
@@ -68,7 +68,7 @@
                     "components": {
                         "showPassword": {
                             "component": "Magento_Customer/js/show-password",
-                            "passwordSelector": "#pass"
+                            "passwordSelector": "#pass,#pass1"
```

Comment:
In fact initialised component should affect only to form from `login.phtml`.
I agree that we have multiple fields with the same id and in this case make sense to use only one field id or bind by field name. Also please use more appropriate names than `pass1`

Reply:
Hello @Den4ik,

First of all, thanks to review this PR. 

I am new in Magento and just trying to contribute here. So to fix the issue we can choose any one option below:

- Use field name - but the name of the field is like this `login[password]`. So how can we pass it in `passwordSelector`?
- Use field class - We can also add a unique class and use that class in `passwordSelector`
- Use field id - This solution is already implemented. As per your suggestion we can change the name of the ID with relevant one. 

Can you please guide me on the first 2 solutions?